app/graphql/mutations/update_school_link.rb

```diff
@@ -0,0 +1,21 @@
+module Mutations
+  class UpdateSchoolLink < GraphQL::Schema::Mutation
+    argument :id, ID, required: true
+    argument :title, String, required: false
+    argument :url, String, required: false
+
+    description 'Update school header/footer/social links'
+
+    field :success, Boolean, null: false
+
+    def resolve(params)
+      mutator = UpdateSchoolLinksMutator.new(context, params)
+      if mutator.valid?
+        { success: mutator.save }
+      else
+        mutator.notify_errors
+        { success: false }
+      end
```

Comment:
This is an old patern. Refer to https://github.com/pupilfirst/pupilfirst/blob/master/app/graphql/mutations/create_course.rb